task/github-app-token/0.1/samples/secret.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: github-app-secret
+type: Opaque
+stringData:
+  private.key: |-
+   -----BEGIN RSA PRIVATE KEY-----
+   MIICXAIBAAKBgQCxiQEUg+S3LmCUDVUaa6q1GM5+phvoL4UXkzr8aa1VZGJCWEEQ
```

Comment:
Thi is a random private key generated from https://travistidwell.com/jsencrypt/demo/ not an actual one (i had some git bots bugging me i have commited a rsa key)